src/Build.UnitTests/FileLogger_Tests.cs

```diff
@@ -321,6 +321,33 @@ public void BasicExistingFileAppend()
         }
 
         /// <summary>
+        /// Logging to a file in a directory that doesn't exists
+        /// </summary>
+        [Fact]
+        public void BasicNoExistingDirectory()
+        {
+            string directory = GetTempDirectoryName();
```

Comment:
Actually there are a few cases where tests in this area are using the current directory instead of the TempProjectDir. Would you like me to have a look at changing those too? If so I'll create another pull request and keep this one issue specific.